packages/chat/src/components/NewFooter/NewFooter.css.ts

```diff
@@ -109,23 +109,32 @@ export const separator = style({
   backgroundColor: COLORS.NEUTRAL_LIGHT[500],
 });
 
-export const extraLinkStyles = style({
-  color: COLORS.NEUTRAL_DARK[100],
-  fontFamily: FAMILY,
-  fontSize: '12px',
-  lineHeight: '17px',
-  textDecorationColor: 'transparent',
-  transition: transition(['color', 'text-decoration-color']),
-  ...hideTextOverflow(),
-  selectors: {
-    '&:hover': {
-      textDecoration: 'underline',
-      color: PALETTE.colors[500],
-      textDecorationColor: PALETTE.colors[500],
-    },
-    '&:active': {
-      color: PALETTE.colors[600],
-      textDecorationColor: PALETTE.colors[600],
+export const extraLinkStyles = recipe({
+  base: {
+    color: COLORS.NEUTRAL_DARK[100],
+    fontFamily: FAMILY,
+    fontSize: '12px',
+    lineHeight: '17px',
+    textDecorationColor: 'transparent',
+    transition: transition(['color', 'text-decoration-color']),
+    ...hideTextOverflow(),
+  },
+  variants: {
+    isLink: {
+      true: {
+        selectors: {
+          '&:hover': {
+            textDecoration: 'underline',
+            color: PALETTE.colors[500],
+            textDecorationColor: PALETTE.colors[500],
+          },
+          '&:active': {
+            color: PALETTE.colors[600],
+            textDecorationColor: PALETTE.colors[600],
+          },
+        },
+      },
+      false: {},
```

Comment:
Sorry for nitpicking on this, but I think you should split this to two styles (without a recipe) -
for example `footerText` and `link`. 

And then you can add one for the text, and both of them together for the link..